Dockerfile

```diff
@@ -25,6 +25,10 @@ ARG BUILDPLATFORM=${BUILDPLATFORM:-amd64}
 FROM --platform=${BUILDPLATFORM} node:16-slim AS superset-node
 
 ARG NPM_BUILD_CMD="build"
+
+RUN apt-get update -q
+RUN apt-get install -yq python3 make gcc g++
```

Comment:
I think best practice would be

```dockerfile
RUN apt-get update -q && apt-get install -yq python3 make gcc g++
```

Reply:
Good call-out, I updated it to follow the same pattern as the rest of the Dockerfile